control/controldisplay/formatter_template.go

```diff
@@ -0,0 +1,169 @@
+package controldisplay
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/turbot/steampipe/control/controlexecute"
+	"github.com/turbot/steampipe/filepaths"
+)
+
+var ErrAmbiguousTemplate = errors.New("ambiguous templates found")
+var ErrTemplateNotFound = errors.New("template not found")
+
+type TemplateFormatter struct {
+	outputExtension string
+	template        *template.Template
+	name            string
+}
+
+func (tf TemplateFormatter) Format(ctx context.Context, tree *controlexecute.ExecutionTree) (io.Reader, error) {
+	reader, writer := io.Pipe()
+	go func() {
+		if err := tf.template.ExecuteTemplate(writer, "outlet", tree); err != nil {
+			writer.CloseWithError(err)
+		} else {
+			writer.Close()
+		}
+	}()
+	return reader, nil
+}
+
+func (tf TemplateFormatter) FileExtension() string {
+	extension := strings.TrimPrefix(tf.outputExtension, ".")
```

Comment:
I thought we were going to leave the dots in the extension as per Go standards

Reply:
done